src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -270,18 +289,48 @@ function SnapCamera(camera) {
 
 var snapShotCamera = null;
 
+let globeIsLoaded = false;
+
+function defer() {
+    const deferedPromise = {};
+    deferedPromise.promise = new Promise((resolve, reject) => {
+        deferedPromise.resolve = resolve;
+        deferedPromise.reject = reject;
+    });
+    return deferedPromise;
+}
+
+const onNextSceneLoadedCb = [];
+
+function waitSceneLoaded() {
+    const deferedPromise = defer();
+    onNextSceneLoadedCb.push(deferedPromise);
```

Comment:
Note that you could even simplify this by doing:
```js
function waitSceneLoaded(viewerDiv) {
    const deferedPromise = defer();
    viewerDiv.addEventListener('globe-built', () => {
       deferedPromise.resolve();
   });
   return deferedPromise;
}
```
That would remove the sketchy `globeIsLoaded` variable as well.

(remember that all these global variables will cause trouble when trying to create 2 GlobeControls, see multiglobe example)